### PR TITLE
Edit vue-router routes to comply with description in Jira

### PR DESCRIPTION
**Pull request by dzldebby**
- /new-student
- /edit-student/:id
- /students (all students)

### CODE REVIEW

File: frontend/src/pages/Students.vue [hunk 1]
```diff
@@ -0,0 +1,356 @@
+
```
Comment (thchia) [inline frontend/src/pages/Students.vue]: This file should not be in this PR, it's part of the Table PR. Remember to start a new branch from master when making new PRs.

File: frontend/src/router.js [hunk 2]
```diff
@@ -17,7 +18,10 @@ const router = new VueRouter({
     { path: "/screening", component: Screening },
     { path: "/matching", component: Matching },
     { path: "/student/:id", component: StudentProfile },
-    { path: "/new", component: NewStudent },
+    { path: "/new-student", component: NewStudent },
+    { path: "/students", component: Students },
```
Comment (thchia) [inline frontend/src/router.js:22]: Technically, you cannot include this until the PR with the `Students.vue` is merged. So you should chase people to review and get that merged! 😃

**Review by thchia — CHANGES_REQUESTED** [review]
Suggest you separate the changes regarding the Students table and the routes.

**Comment by dzldebby** [comment]
> Suggest you separate the changes regarding the Students table and the routes.

Okay, will mark as WIP first until after the Students table is merged.

**Comment by thchia** [comment]
@dzldebby you can rebase and resolve the conflicts on this now.

**Review by thchia — APPROVED** [review]
LGTM

**Review by lauratsui — APPROVED** [review]
Looks good!

**Pull request merged**